charts/pomerium/templates/tls-secrets.yaml

```diff
@@ -105,6 +105,28 @@ type: kubernetes.io/tls
 data:
 {{- end -}}
 
+{{- define "pomerium.proxy.jwtSecretObject" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ template "pomerium.name" . }}
+    helm.sh/chart: {{ template "pomerium.chart" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+  annotations:
+    helm.sh/hook-delete-policy: before-hook-creation
+{{- if .Values.config.forceGenerateTLS }}
+    helm.sh/hook: pre-upgrade
+{{- else if .Values.config.generateTLS }}
+    helm.sh/hook: pre-install
```

Comment:
I _think_ we should have an independent flag for the signing key generation.

Reply:
done